cpu/lm4f120/include/periph_cpu.h

```diff
@@ -121,6 +123,48 @@ typedef enum {
 } adc_res_t;
 #endif /* ndef DOXYGEN */
 
+/**
+ * @brief   Override SPI hardware chip select macro
+ *
+ * As of now, we do not support HW CS, so we always set it to a fixed value
+ */
+#define SPI_HWCS(x)     (UINT_MAX - 1)
+
+/**
+ * @name SPI configuration
+ * @{
+ */
+#define SPI_NUMOF (4)
```

Comment:
Usually we define the number of SPIs according to the number of configurations in the board periph_conf.h as done [here](https://github.com/RIOT-OS/RIOT/blob/master/boards/stm32f4discovery/include/periph_conf.h#L231)